src/user/user.service.ts

```diff
@@ -112,4 +113,19 @@ export class UserService {
 
     return 'Successful';
   }
+
+  public async updateUser(updateUserDto: UpdateUserDto, { user: { id } }: GetUserDto) {
+    const user = await this.userRepository.findOne({ where: { id } });
```

Comment:
is this `id` sent in the request, or automatically got from the firebase user?

We want this logic:
- only the user who is making the request, can update their **own** profile.
- so we need to get the user for the firebase auth token, and verify its their user id that theyre updating, before continuing